nomenclature/cli.py

```diff
@@ -11,6 +11,8 @@
 class PythonLiteralOption(click.Option):
     def type_cast_value(self, ctx, value):
         try:
+            if value is None:
```

Comment:
In the interest of keeping the `try...except` clause as small as possible you could move the `if value is None` check above so that this is done beforehand.